docs/guides/external-node/00_quick_start.md

```diff
@@ -52,17 +52,21 @@ The HTTP JSON-RPC API can be accessed on port `3060` and WebSocket API can be ac
 
 ### System Requirements
 
-This configuration is approximate and should be considered as **minimal** requirements.
+> [!NOTE]
+>
+> This configuration is only for nodes that use snapshots recovery, for requirements for nodes running from DB dump, see
```

Comment:
this is confusing for people with no background. We should mention something along the lines of

> This configuration is only for nodes that use snapshot recovery - they will not have historical data accessible via API. For Archival nodes, please refer to ...

Reply:
I've added some explainations in this "note"